viewer/test-utilities/src/primitives/primitiveAttributes.ts

```diff
@@ -0,0 +1,172 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
```

Comment:
The functions in this file deserves some tsdoc

Reply:
Clearly mark the functions that is meant for outside consumption. Maybe create a `primitives/index.ts` that exports the relevant functions? A lot of exported functions here and not easy to know which functions are relevant for use in the tests.